src/bic/fix-animation.js

```diff
@@ -26,15 +27,17 @@ define(function (require) {
       !!_.intersection(classes, TRANSITIONS).length;
   }
 
-  $(window).on('animationstart', function (event) {
+  $window.on('animationstart', function (event) {
     var timer;
     if (isJQMTransition(event)) {
       // okay, we need to set a 1s timer in case this doesn't finish
       timer = setTimeout(function () {
         c.warn('"animationend" event expected but not observed');
-        $(event.target).trigger('animationend');
+        $window.trigger('animationend');
```

Comment:
aaaah!!!!

i missed one simple thing when reading the animation docs:

target: document, element.

in other words, it doesn't bubble up to the window  where the listener is!